Field Workers are not able to view /trends data on homepage

**Describe the bug**
As a user with the field worker role, I am unable to view the data I should be able to view when navigating to the homepage.

**To Reproduce**
Steps to reproduce the behavior:
1. Login as a user with the field worker role who has some data for their facility
2. Navigate to the homepage
3. Observe lack of data and presence of a 500 error in the developer console

**Expected behavior**
I should be able to view all data on the homepage for which I have the proper permissions.

**Screenshot**
![Screenshot 2023-03-14 at 1 08 57 PM](https://user-images.githubusercontent.com/6718827/225083980-bacf27c2-1a30-48d1-99e0-ba981a9cf858.png)

**Additional context**
Looking at the logs, a likely culprit reveals itself in the stack trace:
```
[2023-03-14 17:02:33 +0000] [60] [ERROR] Error handling request /trends/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/contextlib.py", line 75, in inner
    return func(*args, **kwds)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/app/va_explorer/utils/mixins.py", line 17, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 142, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/va_explorer/home/views.py", line 37, in get
    ) = get_trends_data(request.user)
  File "/app/va_explorer/home/va_trends.py", line 199, in get_trends_data
    VA_TABLE_FIELDS.remove("Id10007")
ValueError: list.remove(x): x not in list
```

--- a/va_explorer/home/va_trends.py
+++ b/va_explorer/home/va_trends.py
@@ -196,20 +196,24 @@ def get_trends_data(user):
         va_table["uncoded"]["1 month"] = len(vas_1_month) - vas_coded_1_month
         va_table["uncoded"]["Overall"] = len(vas_overall) - vas_coded_overall
 
+        # Use a local scope copy of VA_TABLE_FIELDS to avoid modifying the
+        # original which would impact all future requests made by any user and
+        # cause a ValueError: list.remove(x): x not in list
+        user_va_table_fields = list(VA_TABLE_FIELDS)
         if user.is_fieldworker():
-            VA_TABLE_FIELDS.remove("Id10007")
+            user_va_table_fields.remove("Id10007")
 
         # List the VAs that need attention; requesting certain fields and
         # refetching makes this more efficient
         vas_to_address = (
-            user_vas.only(*VA_TABLE_FIELDS)
+            user_vas.only(*user_va_table_fields)
             .filter(causes__isnull=True)[:NUM_TABLE_ROWS]
             .prefetch_related("causes", "coding_issues", "location")
         )
 
         # List the VAs with Indeterminate COD
         vas_with_indeterminate_cod = (
-            user_vas.only(*VA_TABLE_FIELDS)
+            user_vas.only(*user_va_table_fields)
             .filter(causes__cause="Indeterminate")[:NUM_TABLE_ROWS]
             .prefetch_related("causes", "coding_issues", "location")
         )

